README.md

```diff
@@ -62,7 +62,7 @@ config.omniauth :openid_connect, {
 | discovery                    | Should OpenID discovery be used. This is recommended if the IDP provides a discovery endpoint. See client config for how to manually enter discovered values. | no       | false                      | one of: true, false                                 |
 | client_auth_method           | Which authentication method to use to authenticate your app with the authorization server                                                                     | no       | Sym: basic                 | "basic", "jwks"                                     |
 | scope                        | Which OpenID scopes to include (:openid is always required)                                                                                                   | no       | Array<sym> [:openid]       | [:openid, :profile, :email]                         |
-| response_type                | Which OAuth2 response type to use with the authorization request                                                                                              | no       | String: code               | one of: 'code', 'id_token'                          |
+| `response_type`                | Which OAuth2 response type to use with the authorization request                                                                                              | no       | String: code               | one of: 'code', ['id_token', 'token'] <br />**Security note:** Do not use 'token' or 'id_token'. The 'token' (raw OAuth 2.0) MUST NOT be used for the authentication purpose. The 'id_token' is used only for the Self-Issued OpenID Providers. Instead, use ['id_token', 'token'] (the Implicit Flow).                          |
```

Comment:
This looks strange to me. Mixing possible types with strings and arrays is a recipe for confusion. I don't quite understand why this is done this way.

Reply:
Ok, I see https://developers.google.com/identity/protocols/oauth2/openid-connect. I think this should just be `'id_token token`, not some array type.